py/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include sentry_data_schemas/event.schema.json
```

Comment:
@untitaker do you remember what the reason was for organizing schemas per source of truth? (relay generated schemas are in the relay subdirectory)? Asking because I wonder whether we should group them the same way into modules in the python package.

Reply:
This is going to be an important topic to tackle when we will have more schemas. Whether we organize schemas per owner (which is optimal for writing) or conceptually based on the what each schema represents (which is probably better for consuming). 
Now we have one schema only, and for the foreseeable future we will have only one owner and a handful of schemas. We likely can address this later.